### DISCUSSION TITLE
Podman Desktop v1.15 is available 🎉!

### DISCUSSION THREAD

**Discussion by axel7083** [podman-desktop/podman-desktop#10397] [Announcements]
Hello 👋 !

A new release of Podman Desktop is out [v1.15.0](https://github.com/podman-desktop/podman-desktop/releases/tag/v1.15.0)

Download it from the [download section](https://podman-desktop.io/downloads) of the [Podman Desktop website](https://podman-desktop.io/)

Release Notes are available at https://podman-desktop.io/blog/podman-desktop-release-1.15

Full Changelog is available at https://github.com/containers/podman-desktop/releases/tag/v1.15.0

**Comment by YanWenKun** [comment]
Hi there,

I really appreciate your amazing work!
I've been using Podman Desktop daily now.

A quick question: is it just me, or is the download link for Windows missing from this page?
https://podman-desktop.io/downloads/windows

**Reply by YanWenKun** [reply]
Thanks, I see it has links now :-)

**Reply by benoitf** [reply]
it may be related to the GitHub REST API proving a rate limit of 60req/hours

in any case, all files are also available at https://github.com/podman-desktop/podman-desktop/releases/tag/v1.15.0

**Reply by YanWenKun** [reply]
Yes, the links are missing again, it's probably API limit.
